src/pykeen/training/slcwa.py

```diff
@@ -128,10 +128,7 @@ def _mr_loss_helper(
         if _batch_filter is not None:
             positive_scores = positive_scores[_batch_filter]
 
-        return self.model.compute_mr_loss(
-            positive_scores=positive_scores,
-            negative_scores=negative_scores,
-        )
+        return self.model.compute_loss(positive_scores, negative_scores)
```

Comment:
do we still need the helpers here?

Reply:
I think so, because in the training loop's wrapper, some magic stuff happens like preparing the tensors or applying batch filtering/label normalization